README.md

```diff
@@ -105,8 +76,10 @@ other database configurations are in the roles/tpa_single_node/vars/main.yml
 Postgres ssl mode is enabled by default. To disable it please change the following in vars/main.yml file:
```

Comment:
Same here - since above line points to role, does this expect user to modify the per-role settings (would it even be good expectation if its installed as galaxy collection?), or the top-level one.